deploy/charts/klicker-uzh-v2/templates/service-app.yaml

```
@@ -99,3 +99,20 @@ spec:
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.olatApi.service.type }}
+  ports:
+    - port: {{ .Values.olatApi.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
+    {{- include "chart.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: olat-api
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)